rest/route/version.go

```diff
@@ -223,25 +223,25 @@ func (h *versionRestartHandler) Run(ctx context.Context) gimlet.Responder {
 	// RestartAction the version
 	err := dbModel.RestartTasksInVersion(h.versionId, true, MustHaveUser(ctx).Id)
 	if err != nil {
-		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "Database error in restarting version"))
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "restarting tasks in version '%s'", h.versionId))
 	}
 
 	// Find the version to return updated status.
 	foundVersion, err := dbModel.VersionFindOneId(h.versionId)
 	if err != nil {
-		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "Database error in finding version:"))
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "finding version '%s'", h.versionId))
 	}
 	if foundVersion == nil {
 		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
 			StatusCode: http.StatusNotFound,
-			Message:    fmt.Sprintf("version with id %s not found", h.versionId),
+			Message:    fmt.Sprintf("version '%s' not found", h.versionId),
 		})
 	}
 
 	versionModel := &model.APIVersion{}
 	err = versionModel.BuildFromService(foundVersion)
 	if err != nil {
-		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "API model error"))
+		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "converting version to API model"))
```

Comment:
500 and `converting version '%s' to API model`

Reply:
Done.